torchaudio/sox_effects/sox_effects.py

```diff
@@ -179,19 +179,29 @@ def apply_effects_file(
 
 
 @_mod_utils.requires_module('torchaudio._torchaudio')
+@_mod_utils.deprecated('Please migrate to `apply_effects_file` or `apply_effects_tensor`.')
```

Comment:
nit: is it clear in which modules these functions live from the message?

Reply:
That information is automatically encoded. See https://github.com/pytorch/audio/pull/787/files#diff-3ce6065b377e608566858654a3b2c2adL50